HW_8/views.py

```diff
@@ -0,0 +1,53 @@
+from logger import log
+
+
+@log
+def greetings():
+    print('Список персонала')
+
+
+@log
+def show_menu():
+    while True:
+        try:
+            value = int(input(
+                '''\nВыберите команду:\n1 - Просмотреть весь список сотрудников.\n2 - Добавить сотрудника.\n3 - Изменить данные сотрудника.\n4 - Удалить сотрудника.\n0 - Выход.\n'''))
+            if value > 4:
+                print('Введите число от 0 до 4')
+                continue
+            elif value == 0:
+                exit()
```

Comment:
Лучше не использовать эту функцию и таким образом программу не закрывать, но если прям очень хочется, то лучше делать так
```suggestion
                sys.exit()
```
С импортом соответствующего модуля, разумеется.